### ISSUE TITLE
Not Working

### ISSUE THREAD

**Issue by Taqbeel** [Mquinn960/sign-language-help#1]
How does it work. there is no documentation. when come to this help repo it says go to offline trainer for instruction when i go there that says go to help repo.. but not documentation on both sides.
i installed everything in offline trainer which is asked but nothing working...

**Comment by Mquinn960** [comment]
Hello there, what issues are you having currently?

I can't really help if you don't say what the issue is unfortunately.

OpenCV is quite tricky to get set up on your PC, so could it be related to that?

In terms of documentation, each repository has information about the project as does the help repository which is specifically for anyone wanting to run the project(s) in tandem.

The actual code itself is not documented, as this was just a university project - it could definitely use some general docs I agree.

**Comment by Taqbeel** [comment]
No it's not related to that.
Sign language working fine,
Dataset creator working fine,
But offline trainer not working
I installed all dependencies still can't run the project

**Comment by Mquinn960** [comment]
@Taqbeel try pulling the repository fresh, someone just contributed a different version of OpenCV which should fix your issue

**Comment by Taqbeel** [comment]
OK let me try again.
Thank you for the quick response

**Comment by Taqbeel** [comment]
I installed the latest repository.
Ran it. Done with environment variables and can't find the run project
button.

**Comment by Taqbeel** [comment]
There are no run commands in help repository. Just 2 steps. Install and set
environment variables. What else?

**Comment by Mquinn960** [comment]
Hey, are you using IntelliJ IDEA as your IDE? If so there are project settings with configurations available within the repository, otherwise just build the project using Maven and run the Jar like you would any other Java project.

**Comment by Mquinn960** [comment]
@Taqbeel 

![Screenshot_20200712-131345](https://user-images.githubusercontent.com/12242579/87245932-903c2300-c441-11ea-9726-45adf0859d1e.jpg)

**Comment by Taqbeel** [comment]
I am using Android Studio as IDE. I tried to run using maven commands.
But got "no main manifest attributes".

**Comment by Taqbeel** [comment]
Installed IntelliJ Idea. Configured Maven.
Still getting the same error.
No main manifest attributes

**Comment by Mquinn960** [comment]
Hi, you need to define the entrypoint class for the Jar if you run it this way.

Please see https://stackoverflow.com/questions/9689793/cant-execute-jar-file-no-main-manifest-attribute#:~:text=The%20Main%20attribute%20is%20necessary,located%20in%20META-INF%20folder.&text=You%20get%20this%20"no%20main,is%20missing%20from%20the%20MANIFEST.

A little outside the scope of this repo however, as it presumes experience with the tools involved.

**Comment by Taqbeel** [comment]
It is defined.. Still not working

**Comment by Taqbeel** [comment]
Can you please help me to run this.. I am really trying hard to train the
dataset

**Comment by Taqbeel** [comment]
Do you have a demo video of it that how to run it?

**Comment by Mquinn960** [comment]
Hey, if you have IntelliJ, then you should be able to just use one of the existing run configurations to run the code in debug mode.

What errors are you getting? What part of the documentation are you stuck at?

**Comment by Taqbeel** [comment]
I'm done with documentation.. The error I'm getting is..
"no main manifest attribute in Sign-Lang.jar"

Sign-Lang.jar is output file name.

**Comment by Taqbeel** [comment]
Can you please help.
I am really badly stuck at this error..

**Comment by Taqbeel** [comment]
I am getting this error message

**Comment by Mquinn960** [comment]
Hi, I understand you're getting this issue, but no one else is having this problem. Simply follow the instructions in any of the guides, using the correct IDE and build process, and you'll be fine. I'm sorry but I can't help you without more information.

**Comment by Taqbeel** [comment]
I just followed the steps from the your help repository but it didn't help..

**Comment by Taqbeel** [comment]
Just tell me this thing that where to put the dataset folder which I
created with dataset creator.

**Comment by Taqbeel** [comment]
Can you please give a response for this issue?

**Comment by Mquinn960** [comment]
Hi @Taqbeel this issue is closed.

If this issue is related to the Offline Trainer, please raise it here: https://github.com/Mquinn960/offline-trainer/issues **with details**

If this issue is related to the Android app, please raise it here: https://github.com/Mquinn960/sign-language **with details**

However please note this is an Open Source project, offered "as is" - and I can't really help with generic Java issues or stuff to with your IDE.

Kind regards,

Matt
